PayloadPkg/FirmwareUpdate/FirmwareUpdateHelper.c

```diff
@@ -976,6 +977,49 @@ CheckSblConfigDataSvn (
   return EFI_SUCCESS;
 }
 
+/**
+  Perform ACM svn check
+
+  This function will perform svn checks for ACM in flash and
+  ACM in capsule.
+
+  @param[in]  ImageHdr       Pointer to fw mgmt capsule Image header
+
+  @retval  EFI_SUCCESS      SVN check successful.
+  @retval  other            error occurred during firmware update
+**/
+EFI_STATUS
+CheckAcmSvn (
+  IN   EFI_FW_MGMT_CAP_IMAGE_HEADER  *ImageHdr,
+  OUT  UINT8                         *SvnStatus
+  )
+{
+  UINT16                ExistingAcmSvn;
+  UINT16                NewAcmSvn;
+  UINT32                NewAcmHdr;
+
+  if ((ImageHdr == NULL) || (SvnStatus == NULL)) {
+    return EFI_INVALID_PARAMETER;
+  }
+
+  ExistingAcmSvn = GetExistingAcmSvn();
```

Comment:
There would cases where ACM partition will not have ACM bin populated initially. We will need to handle this case.
Incase ACM partition is empty, go ahead and update the ACM.